python/cudf/cudf/_lib/unaryops.pyx

```diff
@@ -143,38 +139,34 @@ def apply_dt_extract_op(incol, outcol, op):
     check_gdf_error(result)
 
 
-def nans_to_nulls(py_col):
+def nans_to_nulls(Column py_col):
     from cudf.core.column import as_column
-
-    py_col = as_column(py_col)
+    from cudf.core.buffer import Buffer
+    from cudf.utils.utils import mask_bitsize, calc_chunk_size
```

Comment:
Are we able to move these `import`s out of this function or do they need to be here for some reason?

Reply:
See above